argocd/overlays/moc-infra/applications/envs/emea/rick/operate-first/opf-kafka.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: argoproj.io/v1alpha1
 kind: Application
 metadata:
-  name: opf-kafka
+  name: opf-kafka-rick
 spec:
   destination:
     name: rick
```

Comment:
@margarethaley there are plans to soon decommission the Rick cluster. Could you please move this application the Morty cluster instead?

Reply:
actually, could you just remove this Argocd Application instead?